ember-welcome-page/rollup.config.mjs

```diff
@@ -10,7 +10,10 @@ const addon = new Addon({
 export default {
   // This provides defaults that work well alongside `publicEntrypoints` below.
   // You can augment this if you need to.
-  output: addon.output(),
+  output: {
+    ...addon.output(),
+    experimentalMinChunkSize: 0,
+  },
```

Comment:
Code can be reverted, once https://github.com/embroider-build/embroider/pull/1449 is merged and is a part of the next release.